### DISCUSSION TITLE
Top three takeaways from our usability testing with visually impaired participants

### DISCUSSION THREAD

**Discussion by amycole501** [uswds/uswds#5661] [Accessibility]
Earlier this year we held five [usability testing sessions](https://github.com/uswds/uswds/wiki/Usability-research-findings-for-Q3-2023) with visually impaired participants. We learned some important lessons:

1. **Add more time than you might expect to your user testing sessions.** Extra time helps you make sure you can adequately observe, including participants’ assistive technology. It also allows time to discuss consent and gives participants and researchers a chance to build rapport. 
2. **No two visually impaired users navigate websites in the same way.** The assistive technology setup and level of experience varied among our participants. Understanding each person's unique approach is crucial, which includes how people adapt to using each site. 
3. **Stay flexible.** If one of our participants had difficulty interacting with a component, we moved to the next scenario in the script. When they couldn’t complete tasks, it showed us that we need to improve the design and development of complex components, and sometimes how we could do so. 

We’re excited to gain more insights from usability testing and learn more about how to make our components and patterns more inclusive and accessible. 

What about your team? Have you conducted user testing? What are some of your top tips you could share with the community?

**Comment by mgifford** [comment]
There are teams like https://makeitfable.com and https://knowbility.org that have made usability testing with people with disabilities their thing. 

This is still relatively rare. 

I would love to see agencies dedicate a % of their usability testing/research budget to people with disabilities. 

Getting more clear direction to do user testing would also be useful. As a vendor, it can be a challenge.

**Reply by amycole501** [reply]
I couldn't agree more. There's only so much a 508 specialist can test as an individual and while some tests can find issues with conformance, that testing process doesn't necessarily ensure usability.  

It would also be great if all user research panels were representative of the general public with 15-20% of the participants identifying as having a disability. Especially as Americans age, it'll be critical to include more diverse panels of participants into user research. I'm hopeful 2024 will usher in some changes in perspective and practice!

**Comment by mariamarrero** [comment]
Thank you @amycole501 for sharing the takeaways from your testing. The USAGov team will be conducting generative research interviews with people with disabilities sometime in January 2024. We will be happy to share our findings here as well. Happy New Year!

**Reply by amycole501** [reply]
How wonderful, we look forward to learning from your work!
